pkg/network/node/networkpolicy.go

```diff
@@ -1078,7 +1088,13 @@ func (np *networkPolicyPlugin) refreshNamespaceNetworkPolicies() {
 				npns.mustRecalculate = true
 			}
 		}
-		if npns.mustRecalculate && npns.inUse {
+		var npnsNotCreated bool
+		if _, ok := np.nsToBeCreated[npns.vnid]; ok {
+			npnsNotCreated = true
+			npns.inUse = true
+			delete(np.nsToBeCreated, npns.vnid)
+		}
+		if npnsNotCreated || (npns.mustRecalculate && npns.inUse) {
```

Comment:
This should all be in `AddNetNamespace`.

Also, need to think about how to make sure `nsToBeCreated` doesnt' leak elements. You should remove elements from it in `DeleteNetNamespace`, and possibly `SyncVNIDRules`

Reply:
correct, added removal code in above mentioned methods to ensure no leaks with `unProcessedVNIDs`.